view/next-project/src/components/fund_information/EditModal.tsx

```diff
@@ -54,7 +54,16 @@ export default function EditModal(props: ModalProps) {
   };
 
   // 担当者を局でフィルタを適用
-  const [bureauId, setBureauId] = useState<number>(1);
+  const [bureauId, setBureauId] = useState<number>(
+    BUREAUS.find((b) => {
+      return (
+        b.id ===
+        props.users.find((u) => {
+          return u.id === props.fundInformation.userID;
+        })?.bureauID
+      );
+    })?.id ?? 1,
+  );
```

Comment:
BUREAUSきてんて探すより、fundInformationのuserIdはわかってるから、それと一致するuserのbureauIdを参照すれば綺麗な気がする！
あと、usersとかの配列からprototype関数を使うとき、中はその単数系を使ってほしい！
例えば
`users.map((user) => ...`
とか
`bureaus.filter((bureau) => ...`
みたいな！

```suggestion
  const [bureauId, setBureauId] = useState<number>(
    props.users.find((user) => user.id === props.fundInformation.userID)?.bureauID || 1,
  );
```